### ISSUE TITLE
Add possibility to configure jwks endpoint path.

### ISSUE THREAD

**Issue by marstran** [auth0/jwks-rsa-java#126]
### Describe the problem you'd like to have solved

My JWK provider is located at `/.well-known/openid-configuration/jwks`, but the library does not let me configure that. It currently only supports this path: `/.well-known/jwks.json`

### Describe the ideal solution

I want to configure the JWK provider path using the `JWKProviderBuilder`, by doing something like this:

```
new JwkProviderBuilder(domain)
    .path("/.well-known/openid-configuration/jwks")
    .build();
```

If a path is not specified in the builder, then the default can still be `/.well-known/jwks.json`.

## Alternatives and current work-arounds

No work-around as far as I know.

**Comment by marstran** [comment]
I just found this comment in a closed issue: https://github.com/auth0/jwks-rsa-java/issues/112#issuecomment-785275421
That might solve my problem.

**Comment by marstran** [comment]
I just found this comment in a closed issue: https://github.com/auth0/jwks-rsa-java/issues/112#issuecomment-785275421
That might solve my problem.

**Comment by lbalmaceda** [comment]
@marstran yes, using the URL provider is what we recommend. You can customize the full URL so it's the most flexible option.
